device/bgpadvanced.yaml

```diff
@@ -0,0 +1,48 @@
+components:
+  schemas:
+    Device.BgpAdvanced:
+      x-status: under-review
+      description: >-
+        Container for BGP advanced settings.
+      type: object
+      properties:
+        hold_time_interval:
+          description: >-
+            Number of seconds the sender proposes for the value of the Hold Timer
+          type: integer
+          default: 90
+        keep_alive_interval:
+          description: >-
+            Number of seconds between transmissions of Keep Alive messages by router
+          type: integer
+          default: 30
+        update_interval:
+          description: >-
+            The time interval at which UPDATE messages are sent to the DUT,
+            expressed as the number of milliseconds between UPDATE messages.
+          type: integer
+          default: 0
+        time_to_live:
+          description: >-
+            The limited number of iterations that a unit of data can experience
+            before the data is discarded.
+          type: integer
+          default: 64
+        tcp_window_size:
```

Comment:
This is a confusing field and does not have dependable repeatable behaviour (  I believe it only works when the connection is initiated and does not work properly when connection is accepted ) . It is not directly linked to BGP protocol operation and can be removed in the initial data model.